app/(pages)/not-found.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link'
+ 
+export default function NotFound() {
+  return (
+    <div>
+      <h2>Not Found</h2>
+      <p>Could not find requested resource</p>
+      <Link href="/">Return Home</Link>

```

Comment:
this applies to most links on the site, but it's especially unclear for this one that it's a link and not just part of the text